src/runtime/contrib/tensorrt/tensorrt_runtime.cc

```diff
@@ -225,10 +266,16 @@ class TensorRTRuntime : public JSONRuntimeBase {
   TensorRTEngineAndContext& GetOrBuildEngine() {
     int batch_size = GetBatchSize();
     int compatible_engine_batch_size = -1;
-    if (FindCompatibleEngine(batch_size, &compatible_engine_batch_size)) {
+    bool find_engine_flag = FindCompatibleEngine(batch_size, &compatible_engine_batch_size);
+    const bool use_int8 = (dmlc::GetEnv("TVM_TENSORRT_USE_INT8", 0) != 0);
+    if (find_engine_flag &&
```

Comment:
I think we can simplify the condition and also make it clearer this way:
```
const bool int8_calibration_not_used_or_not_complete = calibrator_ != nullptr && num_calibration_batches_remaining_ != 0;
if (find_engine_flag && int8_calibration_not_used_or_not_complete) {
  return ...
}
```


If there is a compatible engine already built, we can always use it UNLESS we have finished gathering calibration data and need to build final INT8 engine.